src/util/recalls.ts

```diff
@@ -0,0 +1,110 @@
+import { RecallsSchema } from '@dvsa/cvs-type-definitions/types/v1/recalls';
+import { BearerResponse, MotRecalls, MotSecret } from '../models/motRecalls';
+import logger from './logger';
+
+/**
+ * Search retrieved recall data for an active recall then construct return object.
+ * @param vehicleRecalls
+ * @returns
+ */
+export const filterMotRecalls = (vehicleRecalls: MotRecalls): RecallsSchema => {
+  logger.debug('Filter Recall Response');
+  const foundRecall = vehicleRecalls.recalls.find((recall) => {
+    if (recall.repairStatus === 'NOT_FIXED' && Date.parse(recall.recallCampaignStartDate) < Date.now()) {
```

Comment:
I'd suggest using the Luxton library and specifying the timezone so it's quickly identifiable
```suggestion
    if (recall.repairStatus === 'NOT_FIXED' && DateTime.fromISO(recall.recallCampaignStartDate) < DateTime.now()) {
```

Reply:
Disagree, best to make use of just JS where we can. We are in the lucky position where timezones don't usually matter for us so what we have will be okay